quickwit/quickwit-metastore/src/tests/index.rs

```diff
@@ -157,23 +135,124 @@ pub async fn test_metastore_update_index<
             .deserialize_index_metadata()
             .unwrap();
         assert_eq!(response_metadata.index_uid, index_uid);
-        assert_eq!(
-            response_metadata.index_config.search_settings,
-            new_search_setting
-        );
         assert_eq!(
             response_metadata.index_config.retention_policy_opt,
             loop_retention_policy_opt
         );
         let updated_metadata = metastore
-            .index_metadata(IndexMetadataRequest::for_index_id(index_id.to_string()))
+            .index_metadata(IndexMetadataRequest::for_index_id(
+                index_uid.index_id.to_string(),
+            ))
             .await
             .unwrap()
             .deserialize_index_metadata()
             .unwrap();
         assert_eq!(response_metadata, updated_metadata);
     }
+    cleanup_index(&mut metastore, index_uid).await;
+}
 
+pub async fn test_metastore_update_search_settings<
+    MetastoreToTest: MetastoreService + MetastoreServiceExt + DefaultForTest,
+>() {
+    let (mut metastore, index_uid, index_config) =
+        setup_metastore_for_update::<MetastoreToTest>().await;
+
+    for loop_search_settings in [
+        vec![],
```

Comment:
nit: We tend to use `Vec::new` for `vec![]` and `to_string` instead of `to_owned()`.